chia/full_node/full_node.py

```diff
@@ -168,6 +164,15 @@ def sql_trace_callback(req: str):
                 log.close()
 
             await self.connection.set_trace_callback(sql_trace_callback)
+
+        #configure sqlite database
+        await self.connection.execute("pragma journal_mode=wal")
+        await self.connection.execute("pragma temp_store=memory")
+
+        await self.connection.execute(
+            "pragma synchronous={}".format(db_synchronous_on(self.config.get("db_sync", "auto"), self.db_path))
+        )
```

Comment:
is there a good reason to move this block of code down?

Reply:
The intent was, that the pragma statements get also logged into sql.log (in case sql logging was enabled)